snap/snapcraft.yaml

```diff
@@ -127,52 +127,29 @@ parts:
       # Used when testing an unreleased version of Z-wavejs
       # sed -i 's/"zwave-js".*$/"zwave-js"\: "10.0.5-0-pr-4836-ed10b36"/g' ${SNAPCRAFT_PART_BUILD}/package.json
 
-      # Link the bundled yarn 
-      mkdir "${SNAPCRAFT_PART_BUILD}/bin"
-      cd "${SNAPCRAFT_PART_BUILD}/bin"
-      YARN="$(find ${SNAPCRAFT_PART_BUILD} -name yarn-*.cjs -print0)"
-      ln -s "../${YARN:${#SNAPCRAFT_PART_BUILD}+1}" ./yarn
-      chmod +x ./yarn
-      cd "${SNAPCRAFT_PART_BUILD}"
-
-      # Necessary for the remote build; sometimes.
-      if [ -n "${http_proxy-}" ]; then
-        yarn config set httpProxy "${http_proxy}"
-      fi
-      if [ -n "${https_proxy-}" ]; then
-        yarn config set httpsProxy "${https_proxy}"
-      fi
```

Comment:
These should be there as it is a workaround for the remote build system. But they probably need to be changed for the npm ecosystem. 

But I can look into it tomorrow afternoon 👌 You probably need access to Snapcraft remote builds to debug this.

Reply:
@robertsLando If you happen to know the equivalent commands for npm, let me know. 🫶

Reply:
`npm config set` :)